### ISSUE TITLE
false positive: "Type _EnumMemberT has no attribute 'value'"

### ISSUE THREAD

**Issue by colin-kerkhof** [astral-sh/ty#351]
### Summary

The following code gets flagged with this error: `Type _EnumMemberT has no attribute 'value'`

```python
from enum import Enum


class StatusEnum(int, Enum):
    FIRST = 1

[s.value for s in StatusEnum]
```

### Version

ty 0.0.0-alpha.8 (0474b40e1 2025-05-09)

**Comment by sharkdp** [comment]
Thank you for reporting this. We do not support enums yet. Please follow #183 if you're interested in the progress on enum support.
